src/cljs/owlet_ui/components/branch.cljs

```diff
@@ -4,6 +4,20 @@
             [reagent.core :as reagent :refer [atom]]
             [camel-snake-kebab.core :refer [->kebab-case]]))
 
+(def hover-image (reagent/atom ""))
+
+(defn get-random [images]
+  (get images (rand-int (count images))))
+
+(defn set-hover [images]
+  (let [image (get-random images)]
+    (if (> (count images) 1)
+      ; prevent the same image twice in a row
+      (if (== image @hover-image)
+        (set-hover images)
```

Comment:
@tylerperkins re: the use of `recur` and `while` being more succinct. Agreed. It encourages a style of programming where one emphasizes the intent of the program rather than the names for things in a program (if that makes sense).